### PR TITLE
Log: remove support for external images.

### PR DESCRIPTION
**Pull request by mkrautz**
This change remove support for loading external images.
That is, this change ensures Mumble doesn't load images from  tags in text messages,
comments and channel descriptions via HTTP or HTTPS.

With this change in place, Mumble only supports images that are embedded
in the message via data URLs.

### CODE REVIEW

**Comment by mkrautz** [comment]
Includes https://github.com/mumble-voip/mumble/pull/3167, because they clash.

File: src/mumble/Log.cpp [hunk 1]
```diff
@@ -590,64 +585,26 @@ QVariant LogDocument::loadResource(int type, const QUrl &url) {
 	QImage qi(1, 1, QImage::Format_Mono);
 	addResource(type, url, qi);
 
-	if (url.scheme() != QLatin1String("data") && g.s.iMaxImageSize <= 0) {
-		return qi;
-	}
-
 	if (! url.isValid() || url.isRelative()) {
```
Comment (Kissaki) [inline src/mumble/Log.cpp]: We could drop the isRelative check as we check for data scheme.

File: src/mumble/Log.cpp [hunk 2]
```diff
@@ -590,64 +585,26 @@ QVariant LogDocument::loadResource(int type, const QUrl &url) {
...
-	if (m_onlyLoadDataURLs && url.scheme() != QLatin1String("data")) {
-		shouldLoad = false;
-	}
-
-	if (shouldLoad) {
-		QNetworkReply *rep = Network::get(url);
-		connect(rep, SIGNAL(metaDataChanged()), this, SLOT(receivedHead()));
-		connect(rep, SIGNAL(finished()), this, SLOT(finished()));
+	QNetworkReply *rep = Network::get(url);
+	connect(rep, SIGNAL(metaDataChanged()), this, SLOT(receivedHead()));
```
Comment (Kissaki) [inline src/mumble/Log.cpp]: The method `receivedHead` was removed.

**Comment by Kissaki** [comment]
Related: FR #2185 allow editing welcome message in client

**Comment by Techjar** [comment]
:(

**Pull request merged**